OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -31,6 +31,7 @@
   #
   DEFINE SECURE_BOOT_ENABLE      = FALSE
   DEFINE SMM_REQUIRE             = FALSE
+  DEFINE QEMU_VARS               = FALSE

```

Comment:
Can we make this `QEMU_PV_VARS` or something like that?

Reply:
Sure thing.